library/Icingadb/View/BaseHostAndServiceRenderer.php

```diff
@@ -220,19 +220,56 @@ public function assembleFooter($item, HtmlDocument $footer, string $layout): voi
             );
         }
 
+        if ($item->state->is_acknowledged) {
+            $title = $isService
+                ? sprintf(
+                    $this->translate('Service "%s" on "%s" is acknowledged'),
+                    $item->display_name,
+                    $item->host->display_name
+                )
+                : sprintf($this->translate('Host "%s" is acknowledged'), $item->display_name);
+
+            $statusIcons->addHtml(new Icon(Icons::IS_ACKNOWLEDGED, ['title' => $title]));
+        }
+
+        if ($item->state->in_downtime) {
+            if ($isService) {
+                $message = $item->state->is_handled
+                    ? 'Service "%s" on "%s" is handled by Downtime'
+                    : 'Service "%s" on "%s" is in Downtime';
+
+                $title = sprintf($this->translate($message), $item->display_name, $item->host->display_name);
+            } else {
+                $message = $item->state->is_handled ? 'Host "%s" is handled by Downtime' : 'Host "%s" is in Downtime';
```

Comment:
The object name in the title is superfluous. Please remove it. I think It is sufficient to write `Host/Service is acknowledged/unreachable/flapping...`.